### PR TITLE
Update authentication.cn.md

### PR DESCRIPTION
**Pull request by liuqian1990**

### CODE REVIEW

File: docs/document/content/user-manual/shardingsphere-proxy/configuration/authentication.cn.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ weight = 2
 authentication:
   users:
     root: # 自定义用户名
-      password: root # 自定义用户名
+      password: root # 自定义用户名密码
```
Comment (terrymanu) [inline docs/document/content/user-manual/shardingsphere-proxy/configuration/authentication.cn.md:13]: It is better for 自定义用户名`的`密码

**Review by terrymanu — CHANGES_REQUESTED** [review]

**Comment by terrymanu** [comment]
Can you check english doc too?

**Comment by liuqian1990** [comment]
english doc No error

**Comment by terrymanu** [comment]
Can you update your change which followed by code review suggestion?

**Review by terrymanu — APPROVED** [review]

**Pull request merged**